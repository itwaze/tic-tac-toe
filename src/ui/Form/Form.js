import React from "react";
import styled from "styled-components";

const StyledForm = styled.form`
  width: 50%;
  min-width: 300px;
  border: 1px solid #ccc;
  padding: 24px;
  margin: 0 auto;
  box-sizing: border-box;
  border-radius: 4px;
`;

const Form = (props) => {
  const { children, ...rest } = props;
  return <StyledForm {...rest}>{children}</StyledForm>;
};

export default Form;
