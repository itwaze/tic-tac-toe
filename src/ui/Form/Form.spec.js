import React from "react";
import { render } from "@testing-library/react";

import Form from "./Form";

describe("Form", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(<Form />);
    expect(asFragment()).toMatchSnapshot();
  });
});
