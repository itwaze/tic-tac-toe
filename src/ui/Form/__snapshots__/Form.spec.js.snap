// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should take a snapshot 1`] = `
<DocumentFragment>
  <form
    class="sc-bdfBwQ jYVIre"
  />
</DocumentFragment>
`;
